src/elements/content-picker/Footer.js

```diff
@@ -104,4 +112,4 @@ const Footer = ({
     </footer>
 );
 
-export default Footer;
+export default injectIntl(Footer);
```

Comment:
Hi Marie! You might need to do something along the lines of:
```javascript
export { Footer as FooterBase };
export default injectIntl(Footer);
```
And in the tests, you then `import { FooterBase as Footer }` and mock the `injectIntl()` function

Reply:
smt like here:
https://github.com/box/box-ui-elements/blob/3e358f9aca921c04adb7e60cd14f584001dd38c0/src/components/breadcrumb/__tests__/Breadcrumb.test.js

Reply:
The tests pass without this, is it necessary? Not sure if you were suggesting this when the tests were failing because of incorrect GH setup or if this helps to test the intl specifically

Reply:
I've seen stuff previously where the intl env isn't setup for tests, so they would fail if you don't mock them. Seems to not be the case for BUIE